ops/services/alerts/app_service_metrics/uptime.tf

```diff
@@ -81,3 +83,35 @@ resource "azurerm_monitor_metric_alert" "uptime" {
     ]
   }
 }
+
+resource "azurerm_monitor_metric_alert" "uptime-redirects" {
+  for_each = var.additional_uptime_test_urls
+
+  name                = "uptime-${each.key}"
+  description         = "${each.key} is not responding"
+  resource_group_name = var.rg_name
+  scopes              = [azurerm_application_insights_web_test.uptime[each.key].id, var.app_insights_id]
+  frequency           = "PT1M"
+  window_size         = "PT15M"
+  severity            = var.severity
+  enabled             = contains(var.disabled_alerts, "uptime") ? false : true
```

Comment:
I wonder if this should be "uptime-redirects"? Since we have a separate "uptime" one? 🤔

Reply:
Good catch! Just updated that